submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/tensorflow/TensorFlowConfigEnvGenerator.java

```diff
@@ -19,15 +19,18 @@
 
 package org.apache.submarine.server.submitter.yarnservice.tensorflow;
 
+import java.io.IOException;
+import org.apache.hadoop.net.ServerSocketUtil;
 import org.apache.submarine.commons.runtime.conf.Envs;
 import org.apache.submarine.server.submitter.yarnservice.YarnServiceUtils;
 
 public class TensorFlowConfigEnvGenerator {
 
   public static String getTFConfigEnv(String componentName, int nWorkers,
-      int nPs, String serviceName, String userName, String domain) {
+      int nPs, String serviceName, String userName, String domain) throws IOException {
     String commonEndpointSuffix = YarnServiceUtils
-        .getDNSNameCommonSuffix(serviceName, userName, domain, 8000);
+        .getDNSNameCommonSuffix(serviceName, userName, domain,
```

Comment:
From my understanding, tensorflow config is generated before NM starts TF job worker containers. So the random port still may conflict with the ports on NM node. If the port is generated by NM, then the port conflictions would be resolved.

Reply:
Can we limit the port range to avoid this problem?

Reply:
Thanks @yuanzac and @LinhaoZhu for the review.
make sense. I'm going to update it.

Reply:
https://github.com/apache/submarine/blob/ebad135c1e44bc75f76c41fbaaeece87ed65d144/submarine-server/server-submitter/submitter-yarnservice/src/main/java/org/apache/submarine/server/submitter/yarnservice/YarnServiceJobSubmitter.java#L103-L114
@yuanzac, @liuxunorg 

since we use Yarn Service to submit the job, we couldn't control NM behavior.
so that we couldn't generate port in the NM node.
Please correct me if I'm wrong.

Maybe we could just close this pr since yarn service runtime is deprecated.

Reply:
@tangzhankun Thanks for the reply.
Let me close it.